interface/src/app/signature/participation-type-selector.tpl.html

```diff
@@ -0,0 +1,7 @@
+<div class="alert alert-danger" role="alert" ng-if="$ctrl.errorMessage">
+  {{$ctrl.errorMessage}}
+</div>
+<select name="selectedParticipationType"
```

Comment:
LGTM :+1: - also, I know this isn't in your code but can you change the text to `High-weight genes are those that most strongly influence the signature's activity, and we have found that they often reveal the underlying process or processes captured by the signature.`

The (es) bit looks a little weird, and this way our default load (the HW gene explanation is in some migration, maybe, or is a default?) into a new instance of a server will be something that would be easier to read.